django/publicmapping/redistricting/tasks.py

```diff
@@ -1184,15 +1212,18 @@ def createreport(planid, stamp, request, language=None):
         logger.debug('Getting POST variables and settings.')
 
         info = plan.get_district_info()
-        names = map(lambda i: i[0], info)
+        # names = map(lambda i: i[0], info)   # TODO: Reenable
         nseats = map(lambda i: i[1], info)  # can't do it in the lambda
         nseats = reduce(lambda x, y: x + y, nseats, 0)
         # needs to be a str because of join() below
-        magnitude = map(lambda i: str(i[1]), info)
+        # magnitude = map(lambda i: str(i[1]), info)  # TODO: Reenable
 
         logger.debug('Firing web worker task.')
 
-        dispatcher = HttpDispatchTask()
+        # TODO: This was removed by Celery 4. The recommended solution is to copy-paste the Celery
+        # 3.1 implementation, but it's not clear if we still need this since it seems to be linked
+        # to BARD.
+        #dispatcher = HttpDispatchTask()
```

Comment:
Maybe it's time to just delete this function entirely? It is indeed just for BARD. We'd have to delete the corresponding view and route as well.